web/packages/shared/components/UnifiedResources/UnifiedResources.tsx

```diff
@@ -317,26 +364,38 @@ export function UnifiedResources(props: UnifiedResourcesProps) {
           <Danger>{updatePinnedResourcesAttempt.statusText}</Danger>
         </ErrorBox>
       )}
-      {props.Header(
-        <>
-          {selectedResources.length > 0 &&
-            pinning.kind !== 'hidden' &&
-            (pinning.kind === 'not-supported' ? (
-              <HoverTooltip tipContent={<>{PINNING_NOT_SUPPORTED_MESSAGE}</>}>
-                {$pinAllButton}
-              </HoverTooltip>
-            ) : (
-              $pinAllButton
-            ))}
-        </>
-      )}
+      {props.Header}
       <FilterPanel
         params={params}
         setParams={setParams}
         availableKinds={availableKinds}
         selectVisible={toggleSelectVisible}
         selected={allSelected}
-        shouldUnpin={shouldUnpin}
+        bulkActions={
+          <>
+            {selectedResources.length > 0 && (
+              <>
+                {bulkActions.map(({ key, Icon, text, action }) => {
+                  return (
+                    <ButtonBorder
+                      key={key}
+                      textTransform="none"
+                      onClick={() => action(getSelectedResources())}
+                      css={`
+                        border: none;
+                        color: ${props => props.theme.colors.brand};
+                      `}
+                    >
+                      <Icon size="small" color="branch" mr={2} />
```

Comment:
`color="brand"`?

Reply:
Yes it should be brand but now I'm curious why it works even with this typo